packages/firestore/test/lite/integration.test.ts

```diff
@@ -2038,3 +2045,137 @@
     });
   });
 });
+
+describe('countQuery()', () => {
+  const testDocs = [
+    { author: 'authorA', title: 'titleA' },
+    { author: 'authorA', title: 'titleB' },
+    { author: 'authorB', title: 'titleC' },
+    { author: 'authorB', title: 'titleD' },
+    { author: 'authorB', title: 'titleE' }
+  ];
+
+  it('AggregateQuery and AggregateQuerySnapshot inherits the original query', () => {
+    return withTestCollection(async coll => {
+      const query_ = query(coll);
+
+      const countQuery_ = countQuery(query_);
+      expect(countQuery_.type).to.equal('AggregateQuery');
+      expect(countQuery_.query).to.equal(query_);
+
+      const snapshot = await getAggregateFromServerDirect(countQuery_);
+      expect(snapshot.query.query).to.equal(query_);
+    });
+  });
+
+  it('empty test collection count', () => {
+    return withTestCollection(async coll => {
+      const countQuery_ = countQuery(query(coll));
+
+      const snapshot = await getAggregateFromServerDirect(countQuery_);
+      expect(snapshot.getCount()).to.equal(0);
+    });
+  });
+
```

Comment:
nit: Replace "sample docs" with the number of documents. Also, remove the trailing space character in this string.